### ISSUE TITLE
[FEAT] Add and align success/error messaging

### ISSUE THREAD

**Issue by MikeVedsted** [MikeVedsted/JoinMe#273]
Messaging to the user is currently a mess. It would be nice to have a uniform display component which is used consistently.

This is probably blocked by the error handling on the back end.

**🚀 I'm ready for take off**  
Before submitting, please mark if you:  
- [x] Checked that this feature doesn't already exists  
- [x] Checked that a feature request doesn't already exists  
- [x] Went through the user flow, and understand the impact  
- [x] Made sure the request shows why it is important to users but doesn't exaggerate the value
